File: lib/ce2/models/question.rb
class Question < Tuple
  attribute :stimulus, :string
  attribute :position, :integer
  attribute :supporting_statements, :string
  attribute :image, :string
  attribute :question_set_id, :string
  attribute :explanation, :string
  attribute :name, :string
  attribute :source_info, :string
  attribute :experience_points, :integer
  attribute :deleted_at, :datetime
  attribute :published_at, :datetime
  attribute :spr, :boolean

  relates_to_many :answers do
    Answer.where(Answer.question_id.eq(id))
  end

#  belongs_to :qusetion_set
#  has_many :answers
end

File: public/javascript/app/models/question.js
June.define_set("Question", function(c) { with(c) {
  attributes({
    id: "string",
    stimulus: "string",
    position: "integer",
    supporting_statements: "string",
    image: "string",
    question_set_id: "string",
    explanation: "string",
    name: "string",
    source_info: "string",
    experience_points: "integer",
    deleted_at: "datetime",
    published_at: "datetime",
    spr: "boolean"
  });

  belongs_to("question_set");
}});